src/types.rs

```diff
@@ -30,6 +33,12 @@ pub enum Error {
     /// HTTP request failure
     #[error("HTTP request failure: {0}")]
     Http(#[from] hyper::Error),
+    /// HTTP request failure
+    #[error("HTTP request failure: {0}")]
+    HttpHttp(#[from] hyper::http::Error),
```

Comment:
Not really sure what the best way to handle this was

Reply:
Let's call this `Http` and rename the one currently named `Http` to `Hyper`.